GliaWidgets/SecureConversations/SecureConversations.PendingInteraction.swift

```diff
@@ -0,0 +1,80 @@
+import Combine
+
+extension SecureConversations {
+    final class PendingInteraction: ObservableObject {
+        private(set) var cancellables: Set<AnyCancellable> = []
```

Comment:
There as "sugar" typealias CancelBag, let's decide whether to use or remove it at all.